ReadME.md

```diff
@@ -110,15 +110,15 @@ This project is part of the [GirlScript Summer of Code 2022](https://gssoc.girls
 
 ### Frequently Asked Questions(FAQs) for [GSSoC '22](https://gssoc.girlscript.tech/)
 
-> This project is accepting contributions under [GirlScript Summer of Code 2022](https://gssoc.girlscript.tech/) - three months long Open Source program conducted by [GirlScript Foundation](https://gssoc.girlscript.tech/).
+> This project is accepting contributions under [GirlScript Summer of Code 2022](https://gssoc.girlscript.tech/) - three-month long Open Source program conducted by [GirlScript Foundation](https://gssoc.girlscript.tech/).
 
 [![GSSoC](https://raw.githubusercontent.com/GirlScriptSummerOfCode/MentorshipProgram/master/GSsoc%20Type%20Logo%20Black.png)](https://gssoc.girlscript.tech/)
 
-**1. How to start as a beginner so as to contribute to the project?**
-*There are many beginner level issues to begin with, which include documentation, development and design. Also, feel free to make issues for features as well as for reporting bugs. Participants can claim an issue by commenting that they are interested in it. We are also open to new ideas and suggestions.*
+**1. How to start as a beginner to contribute to the project?**
+*There are many beginner-level issues to begin with, which include documentation, development, and design. Also, feel free to make issues for features as well as for reporting bugs. Participants can claim an issue by commenting that they are interested in it. We are also open to new ideas and suggestions.*
 
 **2. How to communicate with us?**
-*All communication will happen on the Slack channel for this particular projects. Mentors can also be reached through GitHub. Avoid DMs and unnecessary mentions on slack and GitHub both. *Checkout [this](https://github.com/CatsInTech/Rezume/issues) issue for more.**
+*All communication will happen on the Slack channel for this particular project. Mentors can also be reached through GitHub. Avoid DMs and unnecessary mentions on slack and GitHub. *Check out [this](https://github.com/CatsInTech/Rezume/issues) issue for more.**
```

Comment:
These are okay, maybe the last line could replace: `[this](...) issue` with `[these issues](...)`.